twilight-lavalink/CHANGELOG.md

```diff
@@ -13,6 +13,12 @@
 ### Features
 
 - [**breaking**] Reduce size of gateway rest models ([#2168](https://github.com/twilight-rs/twilight/issues/2168))
+- Currently some [Filters](crate::model::outgoing::Filters) are not yet supported.
+Some endpoints such as [Lavalink Info] and [Update Session] have also not yet
+been implemented. Please reach out and open an issue for any missing feature you
+would like to use. The Lavalink V4 port did not add support for any new features
+not previously found in V3 ([#2322](https://github.com/twilight-rs/twilight/pull/2322))
+- TLS feature was added to use secured https and wss protocols, but not was not able to test this path (([#2322](https://github.com/twilight-rs/twilight/pull/2322)))
```

Comment:
Let's get rid of the changelog changes and let us handle that when this is included in a release